nixos/modules/installer/cd-dvd/channel.nix

```
@@ -31,6 +31,7 @@ let
 in
 
 {
+  nix.registry.nixpkgs.flake.outPath = pkgs.path;
```

```suggestion
  nix.registry.nixpkgs.flake.outPath = builtins.path { name = "source"; path = pkgs.path; };
```

For me, my remark about normalizing the path name still applies. Happens if nixpkgs is in a weird location.